posthog/models/feature_flag/feature_flag.py

```diff
@@ -297,6 +297,17 @@ def get_cohort_ids(
 
         return list(cohort_ids)
 
+    def scheduled_changes_dispatcher(self, payload):
+        if "field" not in payload or "value" not in payload:
+            raise Exception("Invalid payload")
+        elif payload["field"] == "active":
+            self.active = payload["value"]
+        elif payload["field"] == "filters":
```

Comment:
not a big fan of using the `filters|active` field to determine what the scheduled change is. It's not specific enough.

For example, very shortly we'll want to add updating multivariate rollout %s (like updating distribution from 10-90 to 40-60 say), and this is also a filters update, but not a release condition.

Would suggest updating these: https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/feature-flags/FeatureFlagSchedule.tsx#L98 to be 'update_status' and `add_release_condition` and renaming `field` to `operation`.